app.py

```diff
@@ -102,6 +109,31 @@ def create_tag():
     conn.close()
     return "Tag(s) Created Successfully", 200
 
+@app.route('/images', methods=['POST'])
+def add_image(): 
+    conn = sqlite3.connect("note.db")
+    cursor = conn.cursor()
+    payload = request.json
+    try: 
+        title = payload.get('title')
+        if title is None: 
+            raise KeyError("Data Entry Requires A Title")
+    except KeyError as error:
+        abort(206, error)
+
+    try: 
+        images = payload.get('image')
```

Comment:
should we validate the image path exists here? i know this is a link to an image, but should it be backend responsibility to throw an error here?

Reply:
I believe it should be both on the frontend and the backend to do check for this. The backend is saving the image path in the database to connect it to a note so I think the backend should check for the image in this regard. I believe the front in should check if the image exists locally in a directory on the user's computer.